phangsPipeline/casaVisRoutines.py

```diff
@@ -887,56 +961,53 @@ def choose_common_res(
     common_res = np.max(ra)*(1.+epsilon)
     return(common_res)
 
+
 def suggest_extraction_scheme(
-    infile_list = [],
-    target_chan_kms = None,
-    restfreq_ghz = None,
-    line = None,
-    vsys_kms=None, vwidth_kms=None, vlow_kms=None, vhigh_kms=None,
-    method = 'regrid_then_rebin',
-    exact = False,
-    ):
+        infile_list=[], target_chan_kms=None, restfreq_ghz=None, line=None,
+        vsys_kms=None, vwidth_kms=None, vlow_kms=None, vhigh_kms=None,
+        method='regrid_then_rebin', exact=False,
+        require_full_line_coverage=False):
     """
     Recommend extraction parameters given an input list of files, a
     desired target channel width, and a preferred algorithm. Returns a
     dictionary suitable for putting into the extraction routine.
     """
 
     # Check inputs
-    
+
     if infile_list is None:
         logging.error("Please specify a list of infiles.")
         raise Exception("Please specify a list of infiles.")
 
     # make sure the input infile_list is a list
-    
+
     if np.isscalar(infile_list):
-        infile_list = [infile_list]    
+        infile_list = [infile_list]
 
     # Require a valid method choice
 
-    valid_methods = ['regrid_then_rebin','rebin_then_regrid','just_regrid','just_rebin']
+    valid_methods = [
+        'regrid_then_rebin', 'rebin_then_regrid', 'just_regrid', 'just_rebin']
     if method.lower().strip() not in valid_methods:
-        logger.error("Not a valid line extraction medod - "+str(method))
+        logger.error("Not a valid line extraction method - "+str(method))
         raise Exception("Please specify a valid line extraction method.")
 
     # Get the line name and rest-frame frequency in the line_list
     # module for the input line
 
     if restfreq_ghz is None:
         if line is None:
-            logging.error("Specify a line name or provide a rest frequency in GHz.")
+            logging.error(
+                "Specify a line name or provide a rest frequency in GHz.")
             raise Exception("No rest frequency specified.")
-        
-        restfreq_ghz = (lines.get_line_name_and_frequency(line, exit_on_error=True))[1]
+        restfreq_ghz = (
+            lines.get_line_name_and_frequency(line, exit_on_error=True))[1]
 
-    # Work out the frequencies at the line edes.
-
-    line_low_ghz, line_high_ghz = lines.get_ghz_range_for_line(
-        restfreq_ghz=restfreq_ghz,  vsys_kms=vsys_kms, vwidth_kms=vwidth_kms,
-        vlow_kms=vlow_kms, vhigh_kms=vhigh_kms)
-    
-    line_freq_ghz = 0.5*(line_low_ghz+line_high_ghz)
+    # # Work out the frequencies at the line edes.
```

Comment:
Why are these now commented out?

Reply:
I think this is because the `line_freq_ghz` variable is no longer being used below. Previously it was used to calculate channel width with the old velocity convention [here](https://github.com/akleroy/phangs_imaging_scripts/blob/dea4d38763b8644e4fdd2469957e4bffe5ab90a9/phangsPipeline/casaVisRoutines.py#L1040)

Reply:
👍